@tompizmor 
```
==> Linting stable/mariadb
[ERROR] templates/: render error in "mariadb/templates/slave-statefulset.yaml": template: mariadb/templates/slave-statefulset.yaml:36:20: executing "mariadb/templates/slave-statefulset.yaml" at <.values.securityCont...>: can't evaluate field securityContext in type interface {}
```

--- a/stable/mariadb/templates/master-statefulset.yaml
+++ b/stable/mariadb/templates/master-statefulset.yaml
@@ -32,10 +32,10 @@ spec:
         release: "{{ .Release.Name }}"
         chart: {{ template "mariadb.chart" . }}
     spec:
-      {{- if .values.securityContext.enabled }}
+      {{- if .Values.securityContext.enabled }}
       securityContext:
-        fsGroup: {{ .values.securityContext.fsGroup }}
-        runAsUser: {{ .values.securityContext.runAsUser }}
+        fsGroup: {{ .Values.securityContext.fsGroup }}
+        runAsUser: {{ .Values.securityContext.runAsUser }}
       {{- end }}
       {{- if eq .Values.master.antiAffinity "hard" }}
       affinity:

--- a/stable/mariadb/templates/slave-statefulset.yaml
+++ b/stable/mariadb/templates/slave-statefulset.yaml
@@ -33,10 +33,10 @@ spec:
         release: "{{ .Release.Name }}"
         chart: {{ template "mariadb.chart" . }}
     spec:
-      {{- if .values.securityContext.enabled }}
+      {{- if .Values.securityContext.enabled }}
       securityContext:
-        fsGroup: {{ .values.securityContext.fsGroup }}
-        runAsUser: {{ .values.securityContext.runAsUser }}
+        fsGroup: {{ .Values.securityContext.fsGroup }}
+        runAsUser: {{ .Values.securityContext.runAsUser }}
       {{- end }}
       {{- if eq .Values.slave.antiAffinity "hard" }}
       affinity:

